<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestOpencv</class>
 <widget class="QMainWindow" name="TestOpencv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestOpencv</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="liveViewLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>901</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyRgbButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Modified Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputMsgButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Console Msg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display (Start)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display (Stop)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="liveViewLineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
